org.kde.itinerary.json

```diff
@@ -14,24 +14,31 @@
     "modules": [
         {
             "name": "poppler",
-            "config-opts": [ "-DENABLE_TESTING=OFF", "-DENABLE_UNSTABLE_API_ABI_HEADERS=ON", "-DENABLE_CPP=OFF", "-DENABLE_GLIB=OFF", "-DENABLE_GOBJECT_INTROSPECTION=OFF", "-DENABLE_UTILS=OFF" ],
+            "config-opts": [
+                "-DENABLE_TESTING=OFF",
+                "-DENABLE_UNSTABLE_API_ABI_HEADERS=ON",
+                "-DENABLE_CPP=OFF",
+                "-DENABLE_GLIB=OFF",
+                "-DENABLE_GOBJECT_INTROSPECTION=OFF",
+                "-DENABLE_UTILS=OFF"
+            ],
             "buildsystem": "cmake-ninja",
             "builddir": true,
             "sources": [
                 {
-                "type": "archive",
-                "url": "https://poppler.freedesktop.org/poppler-21.12.0.tar.xz",
-                "sha256": "acb840c2c1ec07d07e53c57c4b3a1ff3e3ee2d888d44e1e9f2f01aaf16814de7",
-                "x-checker-data": {
-                    "type": "anitya",
-                    "project-id": 3687,
-                    "stable-only": true,
-                    "url-template": "https://poppler.freedesktop.org/poppler-data-$version.tar.gz"
```

Comment:
@flyingcakes85 This is a mistake in the url template, please fix

Reply:
Current pull request can be closed after #11 is merged.